lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -11,6 +11,8 @@
 limitations under the License.
 """
 
+from __future__ import annotations
```

Comment:
What effect does this have here?

Reply:
In this case, it allows you to use `Array[ctypes.c_ubyte]` as a type. In general, it makes type hints behave in this file like they will in Python 3.10:
https://peps.python.org/pep-0563/

Reply:
Makes sense, thanks!